src/content/en/updates/2017/06/credential-management-updates.md

```diff
@@ -75,24 +75,20 @@ asynchronously creates credential objects.
 
 ### Feature detection needs attention
 
-<aside class="warning">
-  <strong>Warning:</strong>
-Because updates to the Credential Management API landed on Chrome 60 contains
-backward incompatible changes, it's important that your implementation won't be
-triggered on older versions. (If you intentionally want to do so, checkout
-<a href="https://docs.google.com/document/d/154cO-0d5paDFfhN79GNdet1VeMUmELKhNv3YHvVSOh8/edit">this
-migration guide doc.</a>
-</aside>
-
-To check if the new Credential Management API is available, check if
-`preventSilentAccess` exists.
+To see if the Credential Management API is available, check if
```

Comment:
Here too, we should refrain from generalities that the CM API *is* available.

Reply:
Can we write:

To see if the Credential Management API for accessing password-based and federated credentials is available, ...